pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -114,52 +106,65 @@ private List<ParticipantWrapperDto> collectData(DDPInstanceDto ddpInstanceDto) {
         List<String> proxyGuids = new ArrayList<>();
 
         for (ElasticSearchParticipantDto elasticSearchParticipantDto : esData.getEsParticipants()) {
+            if (elasticSearchParticipantDto instanceof UnparsedESParticipantDto) {
+                addWrapperToList((UnparsedESParticipantDto) elasticSearchParticipantDto, result);
+            } else {
+                addWrapperToList(elasticSearchParticipantDto, result);
```

Comment:
I would create child class of `ParticipantWrapper` for export feature as well to avoid such implicit if statements, would be more clear in my opinion, but it's personal choice I would say

Reply:
Yeah, once a decision gets made on whether we also want to move the participant list to using unparsed DTOs, we can make a good decision about how best to refactor this